### DISCUSSION TITLE
"Listen" Button for the "from" language

### DISCUSSION THREAD

**Discussion by one-for-all** [ThijmenDam/BarTranslate#31] [Ideas]
Would love to be able to listen to the pronunciation of the word/phrase of the "from" language, just as that for the "to" language, in the front page.

**Comment by ThijmenDam** [comment]
Good idea @one-for-all!

I have created a GitHub issue and labeled it as a feature request. You can view it here: #33. If you have any further questions or ideas, please let me know!
